### ISSUE TITLE
doc: installation instruction do not tell people to add $HOME/go/bin to their $PATH

### ISSUE THREAD

**Issue by davecheney** [golang/go#19613]
The installation instructions tell the user to add`/usr/local/go/bin` to their `$PATH` but make no mention of `$HOME/go` (nee `$GOPATH/go/bin`). This leads to users who believe they have completed all the Go installation instructions, but are confused that commands installed by `go install` or `go get` are not found in their path.

See constabulary/gb#701

**Comment by ALTree** [comment]
To be fair it says:

>  Before rushing off to write Go code please read the How to Write Go Code document, which describes some essential concepts about using the Go tools.

And the linked "How to Write Go Code" doc already suggest adding GOPATH to PATH:

>  For convenience, add the workspace's bin subdirectory to your PATH:
> ```
> $ export PATH=$PATH:$(go env GOPATH)/bin
> ```

In general all this `env` stuff is explained in the "How to Write Go Code", and I'm not sure replicating some of this information in the installation page would be a good idea. 

We already have a "the getting started documents are too wordy" issue. Making them even more wordy because some people don't read info on other pages will make things worse.

**Comment by bradfitz** [comment]
Yeah, I'm inclined to agree with @ALTree. Feel free to argue otherwise. Optimistically closing for now, though.
